dev-python/importlib_resources/importlib_resources-1.0.2-r2.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# This is a backport of Python 3.7's importlib.resources
+PYTHON_COMPAT=( pypy3 python{2_7,3_6} )
+
+inherit distutils-r1
+
+DESCRIPTION="Read resources from Python packages"
+HOMEPAGE="https://importlib-resources.readthedocs.io/en/latest/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+LICENSE="Apache-2.0"
+
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~s390 ~sparc ~x86"
+IUSE="doc test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	$(python_gen_cond_dep 'dev-python/pathlib2[${PYTHON_USEDEP}]' -2)
+	virtual/python-typing[${PYTHON_USEDEP}]
+"
+BDEPEND="
+	doc? ( dev-python/sphinx[${PYTHON_USEDEP}] )
+	test? ( ${RDEPEND} )
+	dev-python/setuptools[${PYTHON_USEDEP}]
+"
+
+# https://gitlab.com/python-devs/importlib_resources/issues/71
+PATCHES=( "${FILESDIR}/${P}-skip-wheel.patch" )
+
+python_prepare_all() {
+	sed -i "/'sphinx.ext.intersphinx'/d" ${PN}/docs/conf.py || die
+	distutils-r1_python_prepare_all
+}
+
+python_compile() {
+	distutils-r1_python_compile
+	if ! python_is_python3; then
+		rm "${BUILD_DIR}/lib/importlib_resources/_py3.py" || die
+	fi
+	if use doc; then
+		sphinx-build ${PN}/docs docs/_build/html || die
```

Comment:
Why do you build it repeatedly for every impl?

Reply:
Oops, that used to be in `python_compile_all` and apparently I moved it for no reason that I can remember. ;-) Thinking of it now, would using `distutils_enable_sphinx` not be a better choice anyway?

Reply:
It would but I don't think it existed when I made the ebuild. I'm going to fix this locally before I merge the PR.

Reply:
Thanks, I'll leave it to you then.